people_options.rb

```diff
@@ -0,0 +1,49 @@
+require './student'
+require './teacher'
+
+class PeopleOptions
+  attr_accessor :people
+
+  def initialize
+    @people = []
+  end
+
+  def list_all_people
+    puts "There is not a person yet...\n" if @people.count.zero?
+
+    @people.each do |person|
+      puts "[#{person.class}] ID: #{person.id} Name: #{person.name} Age: #{person.age}"
+    end
+  end
+
+  # rubocop:disable Metrics/MethodLength
+  def create_person
+    print 'Do you want to create student (1) or teacher (2): '
+    type_of_person = gets.chomp
+    case type_of_person
+    when '1'
+      print 'Name: '
+      name = gets.chomp
+      print 'Age: '
+      age = gets.chomp
+      print 'Parent permission [Y/N]: '
+      pp = gets.chomp
+      pp = false if pp == 'n'
+      pp = true if pp == 'y'
+      student = Student.new(age, name, parent_permission: pp)
+      @people.push(student)
+      puts 'Student created succesfully!'
+    when '2'
+      print 'Name: '
+      name = gets.chomp
+      print 'Age: '
+      age = gets.chomp
+      print 'Specialization: '
+      special = gets.chomp
+      teacher = Teacher.new(special, age, name)
+      @people.push(teacher)
+      puts 'Teacher created succesfully!'
+    end
+  end
+  # rubocop:enable Metrics/MethodLength
```

Comment:
- [ ] Same here, You are not allowed to disable the linter,  Kindly make sure you fix this instead of disabling it 👍🏻 